gbpservice/neutron/nsf/configurator/api/v1/controllers/__init__.py

```diff
@@ -9,18 +9,18 @@
 class ControllerResolver(object):
     """this class send parameter to controller class
     according to query string"""
-    create_network_device_config = controller.Controller(
```

Comment:
I have added nfp directory and also retained nsf directory because removal of that directory was giving conflict in pull request.